libs/message-handler/src/placeholder.test.ts

```diff
@@ -13,7 +13,6 @@ describe('didcomm', () => {
       expires_time: 1516385931,
       body: { messagespecificattribute: 'and its value' },
     });
-    const val = new Message(msg.as_value());
-    return val;
+    expect(msg.as_value()).toBeDefined();
```

Comment:
what does this means?

Reply:
i did not make changes on that branch i only merged the main branch